pkg/splunk/enterprise/searchheadcluster.go

```diff
@@ -557,7 +557,7 @@ func validateSearchHeadClusterSpec(cr *enterprisev1.SearchHeadCluster) error {
 	}
 
 	if !reflect.DeepEqual(cr.Status.AppContext.AppFrameworkConfig, cr.Spec.AppFrameworkConfig) {
-		err := ValidateAppFrameworkSpec(&cr.Spec.AppFrameworkConfig, true)
+		err := ValidateAppFrameworkSpec(&cr.Spec.AppFrameworkConfig, false)
```

Comment:
Should the scope bool be retrieved from the config?  Or just assumed to be `cluster`?

Reply:
I think the default is assumed to be cluster for each CRD and then its checked against the passed config parameter in the CRD(correct me if I am wrong @sgontla )